cmd/config.go

```diff
@@ -46,3 +51,46 @@ func (c *NukeConfig) InBlacklist(searchID string) bool {
 
 	return false
 }
+
+func (c *NukeConfig) resolveDeprecations() error {
+	deprecations := map[string]string{
+		"EC2NatGateway":            "EC2NATGateway",
+		"EC2DhcpOptions":           "EC2DHCPOptions",
+		"EC2VpnConnection":         "EC2VPNConnection",
+		"EC2VpnGateway":            "EC2VPNGateway",
+		"EC2VpnGatewayAttachment":  "EC2VPNGatewayAttachment",
+		"EC2Vpc":                   "EC2VPC",
+		"EC2VpcEndpoint":           "EC2VPCEndpoint",
+		"ECRrepository":            "ECRRepository",
+		"IamGroupPolicyAttachment": "IAMGroupPolicyAttachment",
+		"IamGroup":                 "IAMGroup",
+		"IamInstanceProfileRole":   "IAMInstanceProfileRole",
+		"IamInstanceProfile":       "IAMInstanceProfile",
+		"IamUserGroupAttachment":   "IAMUserGroupAttachment",
+		"IamPolicy":                "IAMPolicy",
+		"IamRolePolicyAttachment":  "IAMRolePolicyAttachment",
+		"IamRole":                  "IAMRole",
+		"IamServerCertificate":     "IAMServerCertificate",
+		"IamUserAccessKeys":        "IAMUserAccessKey",
+		"IamUserPolicyAttachment":  "IAMUserPolicyAttachment",
+		"IamUser":                  "IAMUser",
+	}
+
+	for _, a := range c.Accounts {
+		for resourceType, resources := range a.Filters {
+			replacement, ok := deprecations[resourceType]
+			if !ok {
+				continue
+			}
+			fmt.Printf("deprecated resource type '%s' - replace with '%s'\n", resourceType, replacement)
```

Comment:
You could either do `LogError(fmt.Errorf("depe...` or actually change `LogError` so we have something like `LogError` and `LogErrorString`.

Reply:
@bethge I thought about this. I would love a yellow-ish warning message :-) (ie `LogWarning`)